src/cmds/cpuinfo.c

```diff
@@ -13,6 +13,11 @@
 int main(int argc, char **argv) {
 	struct cpu_info *info = get_cpu_info();
 	printf("CPU Vendor ID:              %s\n", info->vendor_id);
+	printf("CPU Base Frequency (MHz):   %u\n", info->base_freq);
+	printf("CPU Max Frequency (MHz):    %u\n", info->max_freq);
+	printf("CPU FPU Support:            %s\n", info->FPU==1?"true":"false");
```

Comment:
Please add spaces around operators to fit code style (`info->FPU==1?"true":"false"`->`info->FPU == 1 ? "true" : "false"`).